spatialprofilingtoolbox/db/schema_infuser.py

```diff
@@ -66,3 +69,58 @@ def verbose_sql_execute(self, filename_description,
                             **kwargs):
         verbose_sql_execute(filename_description, self.get_connection(),
                             source_package=source_package, **kwargs)
+
+    def link_fraction_views(self):
+        """
+        Transcribe phenotype fraction features in features system
+        """
+        connection = self.get_connection()
+        cursor = connection.cursor()
+        feat_extr_query = """
+            SELECT t1.marker_symbol, t1.measurement_study, t1.average_percent, t1.stratum_identifier, t2.sample
+            FROM fraction_stats  t1
+            LEFT JOIN (
+                SELECT stratum_identifier, MIN(sample) as sample
+                FROM sample_strata
+                GROUP BY stratum_identifier
+            ) t2 ON t1.stratum_identifier = t2.stratum_identifier;
+            """
+        fraction_features = pd.read_sql(feat_extr_query, connection)
+        cursor.close()
+        connection.commit()
+        for study in fraction_features['measurement_study'].unique():
+            study_name = study.replace(' - measurement', '')
+            fraction_features_study = fraction_features[fraction_features.measurement_study == study]
+            with ADIFeaturesUploader(
+                    database_config_file=self.database_config_file,
+                    data_analysis_study=self.insert_new_data_analysis_study(study_name),
+                    derivation_method=self.describe_feature_derivation_method(),
+                    specifier_number=1,
+            ) as feature_uploader:
+                values = fraction_features_study['average_percent'].values
+                # TODO: remove hardcode
+                subjects = fraction_features_study['sample']
+                specifiers_lists = fraction_features_study['marker_symbol'].values
+                for value, subject, specifiers_list in (zip(values, subjects, specifiers_lists)):
+                    feature_uploader.stage_feature_value((specifiers_list,), subject, value)
+
+    def describe_feature_derivation_method(self):
```

Comment:
This is not the right description for the case of the fractions features. This is the description of the proximity metrics feature.